### PR TITLE
更新macOS/Windows上安装并启动Minikube的步骤

### PR DESCRIPTION
**Pull request by jia9iZhang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,28 +24,33 @@ AutoML的部署脚本维护在``./deploy``目录下.
 - Docker
 - Minikube
 
-安装教程:
+如果没有，参考如下安装教程:
 
 **Mac OS:**
 
-- Docker: https://docs.docker.com/desktop/install/mac-install/
```
Comment (free-y) [inline README.md:29]: Perhaps we'd better keep the installation guide of Docker and Minikube

Comment (jia9iZhang) [reply README.md:29]: I don't think it's necessary

Comment (free-y) [reply README.md:29]: Not all contributors in our project are software engineers. There are perhaps also algorithm engineers and UI designers as well as product managers, do you think that whether we need to keep installation guide of Docker and Minikube for them?

Comment (jia9iZhang) [reply README.md:29]: have been added

**Pull request merged**